src/sentry/monitors/serializers.py

```diff
@@ -54,13 +101,17 @@ class MonitorSerializerResponse(TypedDict):
     lastCheckIn: datetime
     nextCheckIn: datetime
     project: ProjectSerializerResponse
+    environments: MonitorEnvironmentSerializerResponse
 
 
 @register(MonitorCheckIn)
 class MonitorCheckInSerializer(Serializer):
     def serialize(self, obj, attrs, user):
         return {
             "id": str(obj.guid),
+            "environment": obj.monitor_environment.environment.name
```

Comment:
does this also need a `select_related`?

Reply:
Yep - although it might be better here to use the `get_attrs` + `prefetch_related_objects` method so that any other endpoints that use this serializer don't have to remember to call `.select_related` on their queryset